deriv.com.conf

```diff
@@ -1,13 +1,16 @@
 server {
     listen 80;
     server_name  _;
+    #### Default Cache headers. This to prevent caching non-existent pages while releasing new versions. Reference -  https://www.gatsbyjs.com/docs/caching/  #######
+    add_header Cache-Control "public, max-age=0, no-cache, no-store, must-revalidate" always;
+    add_header X-CF-IPCOUNTRY $http_cf_ipcountry;
```

Comment:
Why we should return `X-CF-IPCOUNTRY` header? this should add by Cloudflare

Reply:
Nice catch. I think we were snow blind.